tests/lib/client.py

```diff
@@ -78,6 +78,26 @@ def provision_cdn_instance(
             query_string={"accepts_incomplete": accepts_incomplete},
         )
 
+    def provision_migration_instance(
+        self, id: str, accepts_incomplete: str = "true", params: dict = None
+    ):
+        json = {
+            "service_id": "8c16de31-104a-47b0-ba79-25e747be91d6",
+            "plan_id": "739e78F5-a919-46ef-9193-1293cc086c17",
+            "organization_guid": "abc",
+            "space_guid": "123",
+        }
+
+        if params is not None:
+            json["parameters"] = params
+
+        self.put(
+            f"/v2/service_instances/{id}",
```

Comment:
Should this be a `/v3` API call now, or is that not quite ready with this?

Reply:
this is the service broker, not CAPI, so `v2` is current :)

Reply:
Thanks for the clarification! :-)